File: ViewModels/GameViewModel/CurrentGameGameView.cs
using BlackJack.ViewModels.GameViewModel.Enum;
using System;
using System.Collections.Generic;

namespace BlackJack.GameViewModel
{
    public class CurrentGameGameView
    {
        public List<PlayerCurrentGameGameViewItem> PlayersList = new List<PlayerCurrentGameGameViewItem>();
        public PlayerCurrentGameGameViewItem DealerPlayer = new PlayerCurrentGameGameViewItem();
        public GameEnd CheckEndGame;
        public int GameId;
    }
    public class PlayerCurrentGameGameViewItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int CardSum { get; set; }
        public ResultEnumView Result;
        public RoleEnumView Role { get; set; }
        public List<CardCurrentGameGameViewItem> PlayerCards = new List<CardCurrentGameGameViewItem>();
    }
    public class CardCurrentGameGameViewItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Value { get; set; }
        public string Suit { get; set; }
    }
}

File: BlackJack.AngularMVC/models/game-model/CurrentGameGameView.ts
import { ResultEnumView } from "models/enum-model/ResultEnumView";
import { RoleEnumView } from "models/enum-model/RoleEnumView";
import { GameEnd } from "models/enum-model/GameEnd";

export class CurrentGameGameView {
    public playersList: Array<PlayerCurrentGameGameViewItem>;
    public dealerPlayer: PlayerCurrentGameGameViewItem;
    public checkEndGame: GameEnd;
    public gameId: number;
}

export class PlayerCurrentGameGameViewItem {
    public id: number;
    public name: string;
    public cardSum: number;
    public result: ResultEnumView;
    public role: RoleEnumView;
    public playerCards: Array<CardCurrentGameGameViewItem>;
}

export class CardCurrentGameGameViewItem {
    public id: number;
    public name: string;
    public value: number;
    public suit: string;
}